Helper/VarHelper.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Sailthru\MageSail\Helper;
+
+class VarHelper
+
+{
+    private function formatCamelCaseVars($vars)
+    {
+        $ret = [];
+        foreach($vars as $k => $v)
+        {
+            $str = $v[0];
+            for ($i = 0; $i < count($v) - 1; $i++)
+            {
+                $str = $str . ucfirst($v[$i + 1]);
+            }
+            $ret[$k] = $str;
+        }
+        return $ret;
+    }
+    private function formatSnakeCaseVars($vars)
+    {
+        $ret = [];
+        foreach($vars as $k => $v)
+        {
+            $str = $v[0];
+            for ($i = 0; $i < count($v) - 1; $i++)
+            {
+                $str = $str . "_" . $v[$i + 1];
+            }
+            $ret[$k] = $str;
+        }
+        return $ret;
+    }
+    public function getNameKeys($case)
+    {
+        $vars = [
+            'firstname' => ['first', 'name'],
+            'middlename' => ['middle', 'name'],
+            'lastname' => ['last', 'name'],
+        ];
```

Comment:
I think this array is cleaner as a class property as opposed to part of the function itself.